src/pages/contact-us.js

```diff
@@ -51,21 +51,33 @@ const ContactUs = () => (
 
     <ConstrainedWidth>
       <Text size="normal" color="black" maxCharacter="49ch">
-        Follow us on Twitter
+        <p>
+          Follow us on {''}
+          <a
+            className="linkedin-link"
+            target="_blank"
+            href="https://www.linkedin.com/company/neontribe-ltd/"
+            aria-label="This link navigates to the Neontribe LinkedIn page"
```

Comment:
A screen reader will already announce a link, so adding an aria-label here will make the output something like
> follow us on link this link navigates to the Neontribe LinkedIn page

whereas without it, it would be this, which may be clearer?
> follow us on link LinkedIn

Reply:
Agreed!